### PR TITLE
Doc: power evaluated differently fom MATLAB.

### PR DESCRIPTION
**Pull request by PallHaraldsson**
[skip ci]

### CODE REVIEW

**Comment by StefanKarpinski** [comment]
Seems correct to me. Anyone more familiar with Matlab want to review?

File: doc/src/manual/noteworthy-differences.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ may trip up Julia users accustomed to MATLAB:
     in an interactive session. In Julia, unlike MATLAB, `ans` is not set when Julia code is run in
     non-interactive mode.
   * Julia's `struct`s do not support dynamically adding fields at runtime, unlike MATLAB's `class`es.
-    Instead, use a [`Dict`](@ref).
+    Instead, use a [`Dict`](@ref). Dict in Julia isn't ordered.
```
Comment (KristofferC) [inline doc/src/manual/noteworthy-differences.md:73]: Seem unrelated to the PR?

**Comment by KristofferC** [comment]
Just as a note, throwing in a bunch of line break changes and some other small off-hand remarks makes it harder to review the PR than if you simply only add the part that your PR title states.

**Comment by johnnychen94** [comment]
For reference:

```matlab
# matlab
>> 4^3^2

ans =

        4096
```

```julia
# julia
julia> 4^3^2
262144

julia> (4^3)^2
4096
```

File: doc/src/manual/noteworthy-differences.md (hunk 1 above)
Comment (PallHaraldsson) [reply doc/src/manual/noteworthy-differences.md:73]: Dict unordered surprised a colleague of mine, as similar in MATLAB is ordered. Github wants small titles.

**Comment by timholy** [comment]
Thanks!
